Makefile

```diff
@@ -48,6 +48,9 @@ ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=nibiru \
 		  -X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
 		  -X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep)"
 
+ifeq ($(LINK_STATICALLY),true)
```

Comment:
add `LINK_STATICALLY` for wasmd
Ref: https://github.com/CosmWasm/wasmd/blob/master/Makefile#L68